inc/admin-interface.php

```diff
@@ -111,39 +111,59 @@ function admin_notice_old_mu_plugin() {
  */
 function admin_notice_maybe_recommend_higher_max_age() {
 	$current_screen = get_current_screen();
+	$global_warning_shown = current_user_can( 'manage_options' ) ? get_user_meta( get_current_user_id(), 'pantheon_max_age_global_warning_notice', true ) : true;
 
-	if ( apply_filters( 'pantheon_apc_disable_admin_notices', false, __FUNCTION__ ) || 'settings_page_pantheon-cache' !== $current_screen->id ) {
+	if ( apply_filters( 'pantheon_apc_disable_admin_notices', false, __FUNCTION__ ) || $global_warning_shown ) {
 		return;
 	}
 
+	$message = '';
+	$dismissable = true;
 	$max_age_rank = max_age_compare();
 	$current_max_age = get_current_max_age();
+
 	if (
-		$max_age_rank > 0 &&
-		$current_max_age < WEEK_IN_SECONDS &&
-		$current_max_age !== 600
+		$max_age_rank === 0 ||
+		$current_max_age >= WEEK_IN_SECONDS ||
+		$current_max_age === 600
 	) {
-		// If the current max-age value has a rank of 3 or more (10 is the highest), we'll note that it's very low.
+		return;
+	}
```

Comment:
❤️ love me some guard clauses :)

Reply:
yeah, actually simplified this more because I realized that `$max_age_rank === 0` and `$current_max_age >= WEEK_IN_SECONDS` was testing the same thing. the way the max-age rank system works is if it's greater than or equal to the default/recommendation, then it will always be 0 and the # is based on the divergence from that recommendation.